Software/Python/grove_mini_motor_driver/grove_mini_motor_driver.py

```diff
@@ -0,0 +1,249 @@
+from scipy.interpolate import interp1d
```

Comment:
Weird. We've already done that. I've sent a new commit.
It's removed and in its place I've added a custom function which does the same thing.

Cheers!